### PR TITLE
[New] Add recorderjs@0.1.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/mattdiamond/Recorderjs
**Watch 133
  Star 1,711
 Fork 518**
@Amomo could u help me check this PR for #8284 ? thank you!
**this lib isn't maintained anymore, so i regard the current files as v0.1.0 (the same with its package), i checked the git log, the current commit is not so far from [v0.1.0](https://github.com/mattdiamond/Recorderjs/commit/b0cd8f2024a8867194db1e732eb9ce1d88369758), if you think using `08e7abd` as version number is better, i can modify it.**

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR 
- [x] The lib has notable popularity (the author has two lib with high popularity, so we can see its potential.)
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [ ] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

Related issue: https://github.com/mattdiamond/Recorderjs/issues/167

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
The author just updated the readme few weeks ago, I think we can still ask his help for version confirmation, why not?

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by maruilian11** [comment]
@ still get no response from  the author.

**Comment by maruilian11** [comment]
@PeterDaveHello The author still hasn't replied. Can we continue this PR and add auto-update for future usage?

**Comment by PeterDaveHello** [comment]
Try what ever you need to finish it 👍

**Comment by maruilian11** [comment]
@cdnjs/library-reviewer Please help me review this PR, thanks!

**Comment by sufuf3** [comment]
I knew this PR use the latest commit from upstream.
But I think maybe we could use the https://github.com/mattdiamond/Recorderjs/commit/9c4fa710f384333c6dc87f475de5bf808620bffb as v0.1.0. What I consider was the package.json created at this commit and show v0.1.0.

Although the file we needed is better in latest commit because the file was fixed some bug.

Or maybe we can discuss this PR?
Thank you.

**Comment by dakshshah96** [comment]
Hey @sufuf3, I'm sorry but I don't get what you're trying to say. Can you please explain it again?

**Comment by sufuf3** [comment]
@dakshshah96 
@maruilian11 use the commit https://github.com/mattdiamond/Recorderjs/commit/08e7abd99739be6946f19f6806ccb368138f2dd3 (on 21 Jul 2016) to add the file.
But I think the v0.1.0 should use the commit https://github.com/mattdiamond/Recorderjs/commit/47c7e57ba33f12b5a0ea111a545a7a95903d55b5 (on 15 Jan 2016).
Because this commit release v0.1.0 with package.json.

Although there are several commits to fix the file, and the latest is https://github.com/mattdiamond/Recorderjs/commit/08e7abd99739be6946f19f6806ccb368138f2dd3.

**Comment by maruilian11** [comment]
@PeterDaveHello Since the repo is no longer maintained, and some bugs fixed after `package.json` created with v0.1.0, can we add the current version as v0.1.0?

**Comment by PeterDaveHello** [comment]
We should use https://github.com/mattdiamond/Recorderjs/tree/b0cd8f2024a8867194db1e732eb9ce1d88369758 only.

**Comment by maruilian11** [comment]
@cdnjs/library-reviewer @sufuf3 I add files in  https://github.com/mattdiamond/Recorderjs/commit/b0cd8f2024a8867194db1e732eb9ce1d88369758. Could you help me review the PR? Thanks.

**Comment by maruilian11** [comment]
ping @cdnjs/library-reviewer @sufuf3 Thanks!

**Comment by PeterDaveHello** [comment]
This is an old PR, didn't check the checkbox to allow push commits.

**Comment by maruilian11** [comment]
@PeterDaveHello @sufuf3 
checkbox checked
commit rebased

File: ajax/libs/recorderjs/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "version": "0.1.0",
+  "description": "A plugin for recording/exporting the output of Web Audio API nodes",
+  "filename": "recorder.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mattdiamond/Recorderjs.git"
+  },
+  "author": "Matt Diamond",
+  "license": "MIT",
+  "homepage": "http://forked.yannick.io/mattdiamond/recorderjs",
```
Comment (extend1994) [inline ajax/libs/recorderjs/package.json]: This only shows the forked repos, it seems not suitable to be a homepage.

File: ajax/libs/recorderjs/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "recorderjs",
+  "version": "0.1.0",
+  "description": "A plugin for recording/exporting the output of Web Audio API nodes",
+  "filename": "recorder.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mattdiamond/Recorderjs.git"
+  },
+  "author": "Matt Diamond",
```
Comment (extend1994) [inline ajax/libs/recorderjs/package.json]: Would be good if http://matt-diamond.com is added (it's open to the public but the email is only open those who login GitHub) Ref: https://github.com/mattdiamond.

Comment (maruilian11) [inline ajax/libs/recorderjs/package.json]: @extend1994 I think this part is ok. Peter said that we haven't use this field in website yet.

**Comment by maruilian11** [comment]
@cdnjs/library-reviewer I updated the commit, please help me review this PR, thanks!

**Comment by PeterDaveHello** [comment]
@maruilian11 don't you need to mention the source commit in the commit message?

**Comment by maruilian11** [comment]
@PeterDaveHello No. This is the first time that I know I need to add the source in commit msg. I usually don't and mention it only in comments. Like https://github.com/cdnjs/cdnjs/pull/9942

**Comment by maruilian11** [comment]
>Add the reference of the added version please, nobody knows where does it come from in the commit message in the situation there is no version tag in the repo.

I thought about that problem before, it's easy to find the commit in old PRs. So I think it's not a problem to find the source. 

I will add them from now.

**Comment by maruilian11** [comment]
@cdnjs/library-reviewer I add the source, please help me review this PR again, thanks!

**Comment by maruilian11** [comment]
@extend1994 I don't get your meaning. I think the commit is a source.

**Comment by extend1994** [comment]
I mean I think you **traced the commit logs** to figure out the author actually released v0.1.0 and then add the files **according to** that specific commit. I just though the expression `add the files from the commit` is a little weird. Maybe we can see other's opinion on this 😄

**Comment by sufuf3** [comment]
@extend1994 Not sure what you mean?
Do you mean the commit message need to be modified?
If yes, what content is good, in your opinion?

Thank you.

**Comment by maruilian11** [comment]
@sufuf3 @cdnjs/library-reviewer I modified the commit msg.

**Comment by maruilian11** [comment]
@PeterDaveHello Commit msg updated.

**Pull request merged**